src/composables/latestTransactionList.ts

```diff
@@ -37,29 +33,38 @@ export function useLatestTransactionList({ store }: IDefaultComposableOptions) {
   const { balancesTotal } = useBalances({ store });
   const { nodeNetworkId } = useAeSdk({ store });
 
-  const tokens = computed(() => store.state.fungibleTokens.tokens);
-
-  const localPendingTransactionsList = computed(
-    (): ITransaction[] => (store.state.transactions as ITransactionsState)
-      .pending[nodeNetworkId.value!] || [],
-  );
+  const {
+    transactions,
+    fetchTransactions,
+  } = useTransactionList({ store });
 
-  function buildLatestTransactions() {
-    pendingTransactions.value = pendingTransactions.value.filter(({ hash }) => (
-      !fetchedTransactions.value.some((transaction) => transaction.hash === hash)
-    ));
-
-    const transactions: ITransaction[] = [
-      ...pendingTransactions.value,
-      ...fetchedTransactions.value,
-    ];
+  const tokens = computed(() => store.state.fungibleTokens.tokens);
 
-    latestTransactions.value = uniqWith(transactions, (a, b) => (
+  const latestTransactions = computed(() => {
+    const allTransactions = Object.entries(transactions.value)
+      .map(([
+        accountAddress,
+        { loaded, pending }]) => [...(pending[nodeNetworkId.value!] || []), ...(loaded || [])]
```

Comment:
Please fix the formatting here - if you drop a bracket a the first line it should be also dropped on the last making the params more readable
```suggestion
      .map(([
        accountAddress,
        { loaded, pending },
      ]) => [...(pending[nodeNetworkId.value!] || []), ...(loaded || [])]
```